src/PostgREST/ApiRequest.hs

```diff
@@ -141,8 +142,16 @@ userApiRequest schema req reqBody
       ActionInvoke{isReadOnly=True} -> partition (liftM2 (||) (isEmbedPath . fst) (hasOperator . snd)) flts
       _ -> (flts, [])
   flts = [ (toS k, toS $ fromJust v) | (k,v) <- qParams, isJust v, k /= "select", not (endingIn ["order", "limit", "offset", "and", "or"] k) ]
-  hasOperator val = foldr ((||) . flip T.isPrefixOf val) False $ (<> ".") <$> M.keys operators
+  hasOperator val = foldr ((||) . flip T.isPrefixOf val) False ((<> ".") <$> (M.keys operators ++ ["not", show Plain, show Phrase]))
+                    || hasLanguageFts val
   isEmbedPath = T.isInfixOf "."
+  -- handle "?tsv=english.fts.possible" case
+  hasLanguageFts val = case T.splitOn "." val of
+    [_, "fts", _] -> True
+    [_, "fts"]    -> True
```

Comment:
Don't really like this workaround, in hindsight I would've preferred the `fts` syntax to be like:
```http
-- fts with language
GET /tsearch?ts_vector=fts(french).amusant
-- plainto_tsquery
GET /tsearch?ts_vector=plfts.The%20Fat%20Cats
GET /tsearch?ts_vector=plfts(english).The%20Fat%20Cats
-- phraseto_tsquery
GET /tsearch?ts_vector=phfts.The%20Fat%20Rats
GET /tsearch?ts_vector=phfts(english).The%20Fat%20Rats
```
That would make this workaround unnecessary, also it's more consistent with previous operators and the syntax is shorter.

Reply:
Since there hasn't been any release yet, how about if I change the `fts` syntax to the above?

Reply:
what previous opeartors?
how is `fts(french)` shorter then `fts.french`

i don't remember how exactly is the format you implemented but i remember it's modeled after how `not` changes the behaviour of like. `eq.1 vs no.eq.1` and here we might have `fts.test vs french.fts.test`
is it now like `fts.french.test`?

Reply:
Right now we have `not.plain.english.fts.impossible`, with the syntax I mentioned above this would be like `not.plfts(english).impossible`, so the syntax would be shorter for the `phrase/plain` fts modes.

The `not.french.fts.test` would be like `not.fts(french).test`, no gain in length there.

Reply:
this pr needs to be merged to fix the current master, but the format change can be discussed before a release happens

Reply:
Ok. Agreed.